api/Gamepad.json

```diff
@@ -0,0 +1,521 @@
+                "firefox": {
+                  "version_added": "29"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "22"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": "32"
+                },
+                "edge_mobile": {
+                  "version_added": true
+                },
+                "ie_mobile": {
+                  "version_added": false
+                },
+                "opera_mobile": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": true
+                }
+              }
+            }
+          }
+        },
+        "timestamp": {
+          "__compat": {
+            "basic_support": {
+              "support": {
+                "chrome": {
+                  "version_added": "35"
+                },
+                "edge": {
+                  "version_added": true
+                },
+                "firefox": {
+                  "version_added": "29"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": "22"
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": "32"
+                },
+                "edge_mobile": {
+                  "version_added": true
+                },
+                "ie_mobile": {
+                  "version_added": false
+                },
+                "opera_mobile": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": true
+                }
+              }
+            }
+          }
+        },
+        "hand": {
+          "__compat": {
+            "basic_support": {
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false,
+                  "notes": "Enabled in Firefox Nightly and Beta, versions 55 and above. Enabled/disabled by the <code>dom.gamepad-extensions.enabled</code> pref."
```

Comment:
This needs to use a flag object and the Nightly/Beta enabled by default situation can be mentioned in a note.

```
 {
  "version_added": "true",
  "flag": {
    "type": "preference",
    "name": "dom.gamepad-extensions.enabled",
    "value_to_set": "true"
   },
  "notes": "The flag is enabled by default in Firefox Nightly and Beta, versions 55 and above."
 },
```